frontend/src/bundles/create-avatar/components/instruction/instruction.module.css

```diff
@@ -0,0 +1,50 @@
+.flexContainer {
+    flex-direction: column;
+    background-color: white;
+    padding: 30px;
+    border-radius: 10px;
+    align-items: center;
+    gap: 50px;
+    padding-bottom: 80px;
+}
+
+.innerFlex {
+    color: var(--chakra-colors-typography-600);
+    font-size: 14px;
+    justify-content: center;
+    gap: 40px;
+}
+
+.iconContainer {
+    display: flex;
+    gap: 5px;
+    margin-bottom: 30px;
+}
+
+#iconScreenshots {
+    border-radius: 50%;
+    background-color: white;
+    color: var(--chakra-colors-brand-secondary-50);
+}
+
+#iconAvoid {
+    border-radius: 50%;
+    background-color: white;
+    color: var(--chakra-colors-brand-secondary-900);
+}
+
+.textSecondary50 {
+    color: var(--chakra-colors-brand-secondary-50);
+}
+
+.textSecondary900 {
+    color: var(--chakra-colors-brand-secondary-900);
+}
+
+.unorderedList {
```

Comment:
you can use default prop to style it

Reply:
At the moment Button only accepts props: (label, type, variant, isDisables, ClassName, onCLick) so I cant pass maxW as a prop, should I modify the button so it accepts all chakra stylings as direct props?

Reply:
I did it in https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/178 which is this branch's parent.